library/shelve.po

```diff
@@ -25,281 +25,254 @@ msgstr "**Code source :** :source:`Lib/shelve.py`"
 
 #: ../Doc/library/shelve.rst:126
-msgid ""
-"Added the *keyencoding* parameter; previously, keys were always encoded in UTF-8."
+msgid "Added the *keyencoding* parameter; previously, keys were always encoded in UTF-8."
 msgstr ""
-"Ajout du paramètre *keyencoding* ; précédemment, les clés étaient toujours "
-"encodés en UTF-8."
+"Ajout du paramètre *keyencoding* ; précédemment, les clés étaient toujours encodés en UTF-8."
 
 #: ../Doc/library/shelve.rst:130
 msgid "Added context manager support."
 msgstr "Ajout du support des gestionnaires de contexte."
 
 #: ../Doc/library/shelve.rst:136
 msgid ""
-"A subclass of :class:`Shelf` which exposes :meth:`first`, :meth:`!next`, :meth:"
-"`previous`, :meth:`last` and :meth:`set_location` which are available in the "
-"third-party :mod:`bsddb` module from `pybsddb <https://www.jcea.es/programacion/"
-"pybsddb.htm>`_ but not in other database modules.  The *dict* object passed to "
-"the constructor must support those methods.  This is generally accomplished by "
-"calling one of :func:`bsddb.hashopen`, :func:`bsddb.btopen` or :func:`bsddb."
-"rnopen`.  The optional *protocol*, *writeback*, and *keyencoding* parameters "
-"have the same interpretation as for the :class:`Shelf` class."
+"A subclass of :class:`Shelf` which exposes :meth:`first`, :meth:`!next`, :meth:`previous`, :"
+"meth:`last` and :meth:`set_location` which are available in the third-party :mod:`bsddb` "
+"module from `pybsddb <https://www.jcea.es/programacion/pybsddb.htm>`_ but not in other "
+"database modules.  The *dict* object passed to the constructor must support those methods.  "
+"This is generally accomplished by calling one of :func:`bsddb.hashopen`, :func:`bsddb."
+"btopen` or :func:`bsddb.rnopen`.  The optional *protocol*, *writeback*, and *keyencoding* "
+"parameters have the same interpretation as for the :class:`Shelf` class."
 msgstr ""
-"Une sous-classe de :class:`Shelf` qui propose :meth:`first`, :meth:`!next`, :"
-"meth:`previous`, :meth:`last` and :meth:`set_location` qui sont disponible dans "
-"le module tiers :mod:`bsddb` de `pybsddb <https://www.jcea.es/programacion/"
-"pybsddb.htm>`_ mais non dans les autres modules de base de données. L'objet "
-"*dict* passé au constructeur ne doit pas supporter ces méthodes. Cela est "
-"généralement fait en appeleant une des fonctions suivantes : :func:`bsddb."
-"hashopen`, :func:`bsddb.btopen` ou :func:`bsddb.rnopen`. Les paramètres "
-"optionnels *protocol*, *writeback*, et *keyencoding* ont la même signification "
-"que pour la classe :class:`Shelf`."
+"Sous-classe de :class:`Shelf` qui propose :meth:`first`, :meth:`!next`, :meth:`previous`, :"
+"meth:`last` and :meth:`set_location` qui sont disponible dans le module tiers :mod:`bsddb` "
+"de `pybsddb <https://www.jcea.es/programacion/pybsddb.htm>`_ mais non dans les autres "
+"modules de base de données. L'objet *dict* passé au constructeur ne doit pas supporter ces "
```

Comment:
Nous traduisons "to support" par "gérer".
Il me semble que la phrase originale dit l'inverse : l'objet doit (savoir) gérer ces méthodes.